src/test/java/org/commcare/backend/suite/model/test/CaseClaimModelTests.java

```diff
@@ -0,0 +1,33 @@
+package org.commcare.backend.suite.model.test;
+
+import org.commcare.modern.session.SessionWrapper;
+import org.commcare.suite.model.Global;
+import org.commcare.suite.model.MenuDisplayable;
+import org.commcare.suite.model.MenuLoader;
+import org.commcare.suite.model.RemoteQueryDatum;
+import org.commcare.suite.model.SessionDatum;
+import org.commcare.test.utilities.MockApp;
+import org.commcare.util.LoggerInterface;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests for basic app models for case claim
+ *
+ * @author ctsims
+ */
+public class CaseClaimModelTests {
```

Comment:
If the idea here is just to test the parsing, it seems to be happening already as part of [AppStructureTests](https://github.com/dimagi/commcare-core/blob/master/src/test/java/org/commcare/backend/suite/model/test/AppStructureTests.java) today.

Reply:
It is to test parsing for now.

I created a new config because those tests were catching the issues passively. So if, say, someone removed the remote query definition (Say, because they weren't sure what in that suite was part of the actual tests), it wouldn't actually trigger a failure. 

It can be a bit tough with the really dense app files to know when and how changes can be made, so I figured if we were going to extend this functionality at all it probably would be easier to have a smaller, more focused test case.